### ISSUE TITLE
Checkbox is not readable by screen reader arrow keys

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#571]
<dl><dt>Description</dt><dd>Informative (static) content is not readable by a screen reader.
Checkbox is not readable by screen reader using arrow keys.</dd><dt>Impact</dt><dd>Critical</dd><dt>Recommendation</dt><dd></dd><dt>Source code</dt><dd><pre><code><div class="Checkbox is--flex-row"><input id="collapse-box" type="checkbox" aria-describedby="collapse-help"><label class="Field__label" for="collapse-box">Collapse data list</label><div aria-hidden="False" class="Icon Icon--checkbox-checked Checkbox__overlay"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="1 1 22 22" height="24" width="24"><path d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm-9 14-5-5 1.4-1.4 3.6 3.6 7.6-7.6L19 8l-9 9z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div></div></code></pre></dd><dt>Checkpoint</dt><dd>Reading Order (1.3.2.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/DescriptionList</dd></dl>

**Comment by schne324** [comment]
duplicate of #585
